content/en/monitors/notify/variables.md

```diff
@@ -303,6 +202,163 @@ This is the escalation message @dev-team@company.com
+
+**Example**: if your monitor triggers an alert for each `env`, then the variable `{{env.name}}` is available in your notification message.
+
+Variable content is escaped by default. To prevent content such as JSON or code from being escaped, use triple braces instead of double braces, for example: `{{{event.text}}}`.
+
+##### Query group by host
+
+If your monitor triggers an alert for each `host`, then the tag variables `{{host.name}}` and `{{host.ip}}` are available as well as any host tag that is available on this host.
+To see a list of tag variables based on your tag selection, click **Use message template variables** in the **Say what's happening** section.
+Some specific host metadata are available as well:
+
+- Agent Version : {{host.metadata_agent_version}}
+- Machine : {{host.metadata_machine}}
+- Platform : {{host.metadata_platform}}
+- Processor : {{host.metadata_processor}}
+
+##### Query group by a facet
+
+Log monitors, Trace Analytics monitors, RUM monitors and Event monitors can use facets as variables if the monitor is grouped by facet. If a log monitor is grouped by `@facet_key`, use the variable:
+
+```text
+{{ @facet_key.name }}
+```
+
+**Example**: To include group-specific information in a multi alert log monitor group by `@machine_id`:
+
+```text
+This alert was triggered on {{ @machine_id.name }}
+```
+
+If your facet has periods, use brackets around the facet, for example:
+
+```text
+{{ [@network.client.ip].name }}
+```
+
+#### Matching attribute/tag variables
+
+_Available for [Log monitor][2], [Trace Analytics monitor][3] (APM) and [RUM monitor][4]_
+
+To include **any** attribute or tag from a log, a span or a RUM event matching the monitor query, use the following variables :
+
+| Monitor type    | Variable syntax                                         |
+|-----------------|---------------------------------------------------------|
+| RUM             |  `{{rum.attributes.key}}` / `{{rum.tags.key}}`          |
+| Trace Analytics |  `{{span.attributes.key}}` / `{{span.tags.key}}`        |
+| Log             |  `{{log.attributes.key}}` / `{{log.tags.key}}`          |
+
+For any `key:value` pair, the variable `{{log.tags.key}}` renders `value` in the alert message.
+
+**Example**: if a log monitor is grouped by `@error.code`, to include the error message in the notification message, use the variable :
+
+```text
+{{ log.attributes.[error.message] }}
+```
+
+The message renders the `error.message` attribute of a chosen log matching the query, **if the attribute exists**.
+
+<div class="alert alert-info"><strong>Note</strong>: If the picked event does not contain the attribute or the tag key, the variable renders empty in the notification message. To avoid missing notifications, using these variables for routing notification with {{#is_match}} handles is not recommended.</div>
+
+##### First level attributes
```

Comment:
Should we call this as Special attributes or Reserved attributes? First level sounds strange.